### ISSUE TITLE
New file menu should change selection based on typed text

### ISSUE THREAD

**Issue by barmac** [camunda/camunda-modeler#3447]
### Problem you would like to solve

Given I want to create a new diagram via Cmd+N shortcut, when I press "F" afterwards, nothing happens.

https://user-images.githubusercontent.com/28307541/217791615-b7b91e35-1ed8-483c-ba8d-10745b7fd62e.mov

### Proposed solution

In the new file menu, when I type "F", it should select Form. This would follow the existing menu pattern.

https://user-images.githubusercontent.com/28307541/217791835-26f27087-d19a-4144-9614-fe426c8b0e4a.mov

### Alternatives considered

Do nothing.

**Comment by barmac** [comment]
@nikku @philippfromme I believe the pattern is cross-OS, correct?

**Comment by nikku** [comment]
Have not seen this on Linux. If submenu entries exists and accessible through shortcuts, then these are global shortcuts. 

I'd rather support `CTRL/CMD+N` + `F` in the core app (where it is not working, too):

![capture Fz8oCK_optimized](https://user-images.githubusercontent.com/58601/217793706-8aab055a-6a82-4aa9-9b88-15cc632ba4ba.gif)

**Comment by nikku** [comment]
@barmac You probably attached the wrong screen capture to the issue description :arrow_up:. Could you double-check?

**Comment by barmac** [comment]
> I'd rather support CTRL/CMD+N + F in the core app (where it is not working, too):

Yeah I meant that it works in the MacOS menu but does not in the core app.
